docs/incident-management.md

```diff
@@ -0,0 +1,123 @@
+# Incident Management for InstructLab
+
+## Attribution
+Before we start, I would like to mention that this document and process is inspired
+by and influenced heavily from [here][gist] originally authored by [coderanger][ranger].
```

Comment:
Can't we just add this person as a Co-author to this contribution instead of mentioning this?

Reply:
No idea, I figured that there would be _major_ changes to this document as we go forward. This was just a start and figured it would be worth just "tipping the hat to them."

Reply:
I am personally a huge fan of deeply explicit acknowledgements for many reasons, not the least of which is showing people we actually pay attention to the state of the art and best practices other devs/projects are using. JMHO.